fetch.bs

```diff
@@ -4855,6 +4855,72 @@ initially a new {{AbortSignal}} object.
 <a for=Request>request</a>'s
 <a for=request>body</a>.
 
+<dl class=domintro>
+ <dt><code><var>request</var> = new <a constructor lt="Request()">Request</a>(<var>input</var> [,
+ <var>init</var>])</code>
+ <dd>Returns a new <var>request</var> whose <a attribute for=Request>url</a> attribute value is
+ <var>input</var> if <var>input</var> is a string, and <var>input</var>'s
+ <a attribute for=Request>url</a> if <var>input</var> is a {{Request}} object. The optional
+ <var>init</var> argument allows for setting attributes appearing in {{RequestInit}} via object
+ members of the same name.
+
+ <dt><code><var>request</var> . <a attribute for=Request>method</a></code>
+ <dd>Returns <var>request</var>'s HTTP method, which is "<code>GET</code>" by default.
+
+ <dt><code><var>request</var> . <a attribute for=Request>url</a></code>
+ <dd>Returns the URL of <var>request</var> as a string.
+
+ <dt><code><var>request</var> . <a attribute for=Request>headers</a></code>
+ <dd>Returns a {{Headers}} object consisting of the headers associated with <var>request</var>.
+ It should be noted that headers added in the network layer by the user agent will not be accounted
```

Comment:
s/It should be noted that//
(you cannot use terms like "should", "must", or "may" in non-normative text)